src/odemis/acq/stitching/_registrar.py

```diff
@@ -460,6 +460,8 @@ def __init__(self):
         self.shifts_hor = [[None]]
         self.shifts_ver = [[None]]
 
+        self.registered_positions = numpy.array(None)  #list of calculated positions (tuple of 2 floats/None)
```

Comment:
I think `None` would be fine, no need to have an array, if anyway all you want to indicate is that the data is not available.
However, the main thing to update is the comment: It's *not* the calculated positions, and it's *not* a tuple! It's the calculated shifts, as a 3D array...